<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1595</width>
    <height>839</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextEdit" name="codeEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>751</width>
     <height>481</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="errorEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>740</y>
     <width>581</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>710</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>编译信息：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1231</x>
     <y>122</y>
     <width>299</width>
     <height>163</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="tokenButton">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>显示token序列</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="saveButton">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="treeButton">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>显示语法树</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="runButton">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>编译</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="fileButton">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>从本地导入</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="clearallButton">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="linenoEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>61</width>
     <height>481</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="filenameEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>741</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
